.travis.yml

```diff
@@ -472,6 +465,8 @@ before_install:
  - if [ "$PYTHON" == "python3" -a "$TRAVIS_CPU_ARCH" != "arm64" ]; then pyenv global 3.8.1; fi
 #On arm64 (at least) pip is now pip3, so symlink it to pip2 unless we're using python3.
  - if [ "$PYTHON" != "python3" ]; then sudo rm /usr/local/bin/pip && sudo ln -s /usr/local/bin/pip2 /usr/local/bin/pip; fi
+#Switch node to version 17, as newer ones don't work with Ubuntu 18.04 and some older ones are too old
+ - if [ "$TASK" = "jshint" ]; then nvm install 17 && nvm use 17; fi
```

Comment:
You should install either v16 or v18, as those are LTS releases. See https://github.com/nodejs/release#release-schedule:

> Odd-numbered release lines are not promoted to LTS - they will not go through the 'Active LTS' or 'Maintenance' phases.

Reply:
Thanks @FloEdelmann . I didn't realise they'd still be doing updates for it. We can't use 18 unfortunately as it doesn't work on 18.04.